airflow/executors/base_executor.py

```diff
@@ -332,10 +333,17 @@ def slots_available(self):
             return sys.maxsize
 
     @staticmethod
-    def validate_command(command: List[str]) -> None:
-        """Check if the command to execute is airflow command"""
+    def validate_command(command: List[str]) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
nit: it might be more intuitive to call it `validate_airflow_tasks_run_command` as it is tight to it.

Reply:
Crossed my mind too :)